content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: 'Create GDS from Tinytapeout Git Templates'
+linkTitle: 'Template to GDS'
+description: 'preparing your design for submission using the TinyTapeout GitHub template'
+weight: 60
+---
+
+
+This guide will walk you through the process of preparing your design for submission using the TinyTapeout GitHub template. We’ll cover the necessary steps to convert your design into the ASIC files required for chip manufacturing and also view your files with the 3d viewer.
+If you don't already have a GitHub account, please create one before proceeding.
+
+## Pre-requisites
+
+**Check Wokwi Project**: The digital design that you want to tapeout can be created following this [video tutorial](https://tinytapeout.com/digital_design/wokwi/). Note that your wokwi project should comply with the [Tiny Tapeout Wokwi template](https://wokwi.com/projects/354858054593504257).
+
+**Github Account**:  If you don’t have a GitHub account, sign up for one from [here](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
+
+
+## Access the GitHub Template
+
+1. **Navigate to the Template Repository**: 
```

Comment:
In general the formatting is badly done, there are too many combinations of bold, lists, headings. Stick to the ordered headings like # ## ### etc. Follow how the other guides are done.